rules/windows/persistence_local_scheduled_task_creation.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2020/02/18"
+maturity = "production"
+updated_date = "2021/03/15"
+
+[rule]
+author = ["Elastic"]
+description = "A scheduled task can be used by an adversary to establish persistence, move laterally, and/or escalate privileges."
+false_positives = ["Legitimate scheduled tasks may be created during installation of new software."]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Local Scheduled Task Creation"
+risk_score = 21
+rule_id = "afcce5ad-65de-4ed2-8516-5e093d3ac99a"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence with maxspan=1m
+  [process where event.type != "end" and
+    (process.name : ("cmd.exe", "wscript.exe", "rundll32.exe", "regsvr32.exe", "wmic.exe", "mshta.exe","powershell.exe",
+                     "pwsh.exe", "WmiPrvSe.exe", "wsmprovhost.exe", "winrshost.exe") or
+    process.code_signature.trusted == false)] by process.entity_id
+  [process where event.type == "start" and
+    (process.name : "schtasks.exe" or process.pe.original_file_name == "schtasks.exe") and
+    process.args : ("/create", "-create") and process.args : ("/RU", "/SC", "/TN", "/TR", "/F", "/XML") and
+    /* exclude SYSTEM SIDs - look for task creations by non-SYSTEM user */
+    not user.id : ("S-1-5-18", "S-1-5-19", "S-1-5-20")] by process.parent.entity_id
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1053"
+name = "Scheduled Task/Job"
+reference = "https://attack.mitre.org/techniques/T1053/"
+
+[[rule.threat.technique.subtechnique]]
+id = "T1053.005"
+name = "Scheduled Task"
+reference = "https://attack.mitre.org/techniques/T1053/005/"
+
+
+
```

Comment:
@bm11100 Extra line need to be removed?

Reply:
the linter always does 3 new lines and a new line at the end. we can manually modify if we want, but shouldn't affect anything. eventually i think we will nest it  
```
[[rule.threat]]
framework = "MITRE ATT&CK"

  [rule.threat.tactic]
  name = "Execution"
  id = "TA0002"
  reference = "https://attack.mitre.org/tactics/TA0002/"

  [[rule.threat.technique]]
  id = "T1059"
  name = "Command and Scripting Interpreter"
  reference = "https://attack.mitre.org/techniques/T1059/"

    [[rule.threat.technique.subtechnique]]
    name = "Unix Shell"
    id = "T1059.004"
    reference = "https://attack.mitre.org/techniques/T1059/004/"
```